docs/reference/faq.md

```diff
@@ -0,0 +1,190 @@
+# Known issues
+
+This page lists some known limitations, issues and frequently asked questions.
+
+
+(faq_arbitrary_package_names)=
+
+## Is it possible to use arbitrary package names?
+
+No, package names must be the same as the package names in the archive, so that
+there's a single namespace to remember and respect.
+
+
+(faq_available_ubuntu_versions)=
+
+## Which Ubuntu versions have package slices?
+
+The slice definitions are located at the {{chisel_releases_repo}}. The
+`ubuntu-XX.YY` branches in that repository contains the slice definitions for
+the corresponding Ubuntu releases.
+
+If you find a specific release missing, let the maintainers know by [creating a
+new issue] in the {{chisel_releases_repo}}.
+
+
+(faq_non-ubuntu_archives)=
+
+## Is it possible to use non-Ubuntu archives?
+
+Not yet. The supported archives are described at
+{ref}`chisel_yaml_format_spec_archives`.
+
+
+(faq_ppa)=
+
+## Is it possible to use PPAs?
+
+Not yet.
+
+
+(faq_file_ownership_preserved)=
+
+## Is file ownership preserved?
+
+No, Chisel does not yet preserve the owner UID:GID of files extracted from
+packages.
+
+
+(faq_per-arch_dependency)=
+
+## Does Chisel support per-architecture slice dependency?
+
+Although a package may require additional packages based on the architecture,
+Chisel currently does not support listing per-architecture slices in
+{ref}`slice_definitions_format_slices_essential`.
+
+See [Issue 93].
+
+
+(faq_reproducible_output)=
+
+## Does Chisel support reproducible output?
+
+Chisel always fetches the latest version of a package from the archives. Thus,
+the root file systems Chisel produces in subsequent executions may not be
+identical if a package has changed in the meantime.
+
+Chisel also does not support pinning package versions, nor does it support the
+[snapshot service].
+
+See [Issue 154].
+
+
+(faq_refer_same_path)=
+
+## Can multiple slices refer to the same path?
```

Comment:
hum this is more of an explanation for a feature that's to come. I'd remove for now

Reply:
Removed.